### PR TITLE
EOS-22091: Permission fix for message log file

### PR DESCRIPTION
**Pull request by suryakumar1024**
Signei-off-by: suryakumar.kumaravelan <suryakumar.kumaravelan@seagate.com>

# Problem Statement
- MessageBus Log file permission issue. Non admin user can't able to access or write message_log.py
- Fixed this issue by changing permissions for log directory and log file. 

```
After permission change
For file permission:
-rw-rw-r-- 1 root root 14247 Jul  5 03:53 message_bus.log

Directory permission:
drwxrwxr-x 2 root root 4096 Jul  4 14:02 message_bus
```

# Design
-  For Bug describe the fix here. 
-  For Feature, Post the link to the solution page on the confluence CORTX Foundation Library 

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: Y
-  Confirm All CODACY errors are resolved [Y/N]: Y

# Testing 
- [ ] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: 
- [ ] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: 
- [ ] Confirm Testing was performed with installed RPM [Y/N]:  
```
Following steps were performed:

1. Custom VM deployment using Custom CI build.
2. Permissions verified,
3. unstall Cortx-utils rpm, deleted directories
4. Reinstalled rpm
5. permissions verified.

```

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [x] Is there a change in filename/package/module or signature [Y/N]: 
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]
- [x] Jira is updated
- [x] Check if the description is clear and explained. 
- [x] Check Acceptance Criterion is defined. 
- [x] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

File: py-utils/src/utils/message_bus/message_bus.py [hunk 1]
```diff
@@ -36,6 +38,9 @@ def __init__(self):
         log_level = Conf.get('config_file', 'utils>log_level', 'INFO')
         Log.init('message_bus', '/var/log/cortx/utils/message_bus',
             level=log_level, backup_count=5, file_size_in_mb=5)
+        os.chmod("/var/log/cortx/utils/message_bus", stat.S_IWRITE)
```
Comment (sachinpunadikar) [inline py-utils/src/utils/message_bus/message_bus.py:41]: Permissions on directory should be 775 -> (stat.S_IRWXU | stat.S_IRWXG | stat.S_IROTH | stat.S_IXOTH)

File: py-utils/src/utils/message_bus/message_bus.py [hunk 2]
```diff
@@ -36,6 +38,9 @@ def __init__(self):
...
+        os.chmod("/var/log/cortx/utils/message_bus/message_bus.log",
```
Comment (sachinpunadikar) [inline py-utils/src/utils/message_bus/message_bus.py:42]: Permissions on file should be 664 -> (stat.S_IREAD | stat.S_IWRITE | stat.S_IRGRP | stat.S_IWGRP | stat.S_IROTH )

**Review by sachinpunadikar — CHANGES_REQUESTED** [review]
Please work on the comments. Expected permissions are 775 on directory & 664 on file

**Comment by sachinpunadikar** [comment]
Add JIRA id to the title
Fill up the template

File: py-utils/src/utils/message_bus/message_bus.py (hunk 1 above)
Comment (suryakumar1024) [reply py-utils/src/utils/message_bus/message_bus.py]: Updated in recent patch.

File: py-utils/src/utils/message_bus/message_bus.py (hunk 2 above)
Comment (suryakumar1024) [reply py-utils/src/utils/message_bus/message_bus.py]: Updated in recent patch.

**Review by rahul-tripathi-git — APPROVED** [review]
Looks good to me.

**Review by sachinpunadikar — APPROVED** [review]
+1

**Pull request merged**